torchxrayvision/baseline_models/chestx_det/__init__.py

```diff
@@ -0,0 +1,118 @@
+import sys, os
+import json
+import pathlib
+import torch
+import torch.nn as nn
+import torchvision
+from collections import OrderedDict
+from .ptsemseg.pspnet import pspnet
+
+
+def _convert_state_dict(state_dict):
+    """Converts a state dict saved from a dataParallel module to normal 
+       module state_dict inplace
+       :param state_dict is the loaded DataParallel model_state
+    
+    """
+    new_state_dict = OrderedDict()
+    for k, v in state_dict.items():
+        name = k[7:]  # remove `module.`
+        new_state_dict[name] = v
+    return new_state_dict
+
+
+class PSPNet(nn.Module):
+    """
+    https://github.com/Deepwise-AILab/ChestX-Det-Dataset
+
+    @article{Lian2021,
+        title = {{A Structure-Aware Relation Network for Thoracic Diseases Detection and Segmentation}},
+        author = {Lian, Jie and Liu, Jingyu and Zhang, Shu and Gao, Kai and Liu, Xiaoqing and Zhang, Dingwen and Yu, Yizhou},
+        doi = {10.48550/arxiv.2104.10326},
+        journal = {IEEE Transactions on Medical Imaging},
+        url = {https://arxiv.org/abs/2104.10326},
+        year = {2021}
+    }
+
+
+    """
+
+    def __init__(self):
+
+        super(PSPNet, self).__init__()
+        
+        self.transform = torchvision.transforms.Compose([
+            torchvision.transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
+        ])
+        
+        self.targets = ['Left Clavicle', 'Right Clavicle', 'Left Scapula', 'Right Scapula',
```

Comment:
I think it would be cleaner to make this a property:

```
def __init__(self):
    ...
    self._targets = ['Left Clavicle', 'Right Clavicle', 'Left Scapula', 'Right Scapula',...]
    ...

@property
def targets(self):
    return self._targets
```

This allows the user to view `self.targets` but not manipulate the list in any way. Since it will be used by the user to index the segmentations, any accidental manipulation of that list could be nasty.